README.md

```diff
@@ -1,2 +1,12 @@
-# hw01
-Repository for homework 1
+#Carman Fowler Biography
+##Personal
+###I am from Camden, South Carolina.
+###![Camden Sign](http://maineanencyclopedia.com/wp-content/uploads/camden01.jpg)
+###My hobbies include: 
+###1. *Creative writing*
+###2. Going on *ghost tours*
+###3. Playing *Dungeons and Dragons*. 
+###This is my first year in Chicago and I haven't seen a lot of the city but my favorite place to go is the Art Institute. 
+##Research
+###I am in the **MAPSS** program and I work with **Dr. Ayelet Fishbach** in the **Booth School**. My research interests involve moral decision making, cognitive dissonance, and effects of culture and identity. 
```

Comment:
Great job here, the text you sought to bold appeared properly on the GitHub page.